package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "state-trooper",
-  "version": "0.3.0",
+  "version": "0.4.0",
   "description": "Application State Manager",
   "main": "index.js",
   "scripts": {
-    "package": "browserify index.js -t babelify --outfile statetrooper.bundle.js",
-    "test": "./node_modules/mocha/bin/mocha --compilers js:babel/register test/**_test.js"
+    "test": "./node_modules/mocha/bin/mocha --compilers js:babel/register test/**_test.js",
```

Comment:
FYI you can just call `mocha` and npm resolves it to `node_modules/.bin/mocha`
So this could be shorted a bit:
`"test": "mocha --compilers js:babel/register test/**_test.js"`